index.html

```diff
@@ -59,71 +56,144 @@ <h1 class="header-title">
+              >
+              <h2 class="description-section__title">Shop for your favorites <br>meal as e dey hot.</h2>
+              <p class="description-section__description">Shop for your favorite meals or drinks <br>and enjoy while doing it.</p>
             </div>
-            <img class="interface-app__img" src="images/Rectangle (2).png" alt="interface-app__img">
           </div>
+        </div>
       </div>
     </section>
-    <section class="payment-info">
-      <div class="container">
-        <div class="payment-info__description">
-          <img class="payment-info__img" src="images/Rectangle 3.png" alt="payment-info__img">
-            <div class="payment-info__item">
-              <h3 class="payment-info__checkout">Checkout</h3>
-              <h1 class="payment-info__advantages">When you done check out<br> and get it delivered.</h1>
-              <h4 class="payment-info__deliver">When you done check out and get it<br> delivered with ease.</h4>
+    <section class="description-section">
+      <div class="sm-container">
+        <div class="description-section__columns">
+          <div
+            class="description-section__column description-section__column-img"
+          >
+            <img
+              src="images/Rectangle 3.png"
+              alt="description-section__img"
+            />
+          </div>
+          <div
+            class="description-section__column description-section__column-info"
+          >
+            <div>
+              <span class="description-section__benefit-text"
+                >Checkout</span
+              >
+              <h2 class="description-section__title">
+                When you done check out <br>and get it delivered.
+              </h2>
+              <p class="description-section__description">
+                When you done check out and get it <br>delivered with ease.
+              </p>
             </div>
+          </div>
         </div>
       </div>
     </section>
-    <section class="section">
+    <section class="section-benefits">
       <div class="container">
-        <h1 class="section-top">Download the app now.</h1>
-        <h2 class="section-row">Available on your favorite store. Start your premium experience now</h2>
-        <div class="section-buttons">
-          <button class="header-buttons__btn header-buttons__playstore">
-            Playstore
-          </button>
-          <button class="header-buttons__btn header-buttons__appstore">
-            App store
-          </button>
+        <div class="section-benefit__text">
+          <h2 class="section-benefit__dwn">Download the app now.</h2>
+          <h3 class="section-benefit__description">
+            Available on your favorite store. Start your premium experience now
+          </h3>
+          <div class="section-buttons">
+            <button class="header-buttons__btn header-buttons__playstore"><a class="link-btn" href="https://play.google.com/store">
```

Comment:
1) It's not header-buttons so let's change classes to generic so we can use it in footer and header
2) Use only tag <a></a> it's bad practice to put link in the button or conversely.
<a class="link-btn cta-buttons__btn cta-buttons__playstore" href="https://play.google.com/store">Playstore</a> that's all